### ISSUE TITLE
[Error] Can't access external drive

### ISSUE THREAD

**Issue by Firlaev-Hans** [89luca89/distrobox#729]
**Describe the bug**
I have an external HDD mounted on my computer. But while the mountpoints inside `/run/media` are visible from within the distrobox, trying to view any of their contents returns Permission Denied.  The folders are all shown as being owned by `nobody` and `nogroup`, with permissions `rwx------` (it's actually `rwxr-xr-x` on the host).

**To Reproduce**
Mount an external drive on your computer and try to access the drive from inside the distrobox

**Expected behavior**
I should be able to access the external drive from the toolbox

**Desktop (please complete the following information):**
 - Host: PostmarketOS Edge (Alpine based) on aarch64
 - Guest: Ubuntu 22.04
 - Distrobox 1.4.2.1 with rootless podman

**Comment by 89luca89** [comment]
Hi @Firlaev-Hans 

Seems strange that permissions are `0700` in the container, and different on the host, this shouldn't happen
Distrobox quite simply bind mounts the directories, so everything should be the same

**Comment by Firlaev-Hans** [comment]
Alright, so I feel kinda stupid because I just tried again and... it works?
When I first enter the distrobox contain, I get a warning:
`WARN[0000] "/" is not a shared mount, this could cause issues or missing mounts with rootless containers`
So I googled that and made `/` a shared mount. I thought I did that only *after* I had trouble accessing my drive, to try and fix it, I guess I was misremembering things. Because now I just ignored the warning for once, and now it all works.
So, uh, nevermind. Sorry.
